### DISCUSSION TITLE
記事一覧ページでの2カラム表示はユーザー選択式で復活させられないか

### DISCUSSION THREAD

**Discussion by yamadashy** [increments/qiita-discussions#192] [要望・アイデア・不具合]
# 概要

[今後のQiitaのUIのアップデート計画について](https://blog.qiita.com/ui-update-plan-for-2022/)

こちらの記事にて
> 「端末の解像度に合わせてコンテンツ幅を調整できるので、自由に選べるようになって良い」というご意見をいただいた一方で、
「視線の移動が複雑化し、見づらくなった」というご意見も多数いただきました。
我々Qiita運営としては、よりよいコンテンツへのアクセス体験を提供をしていきたいと考えています。
つきましては、皆様のフィードバックを真摯に受け止め記事カードを1列にいたします。

とあり、トップページなどの記事一覧の2カラム表示を廃止されたようですが、
個人的には1画面に表示される記事数が多くて見やすかったので、ユーザーが選択できる形で復活させるなどをご検討いただきたいです。

# イメージ
雑ですが下の画像のように右上に切り替えボタンがあるイメージです。
- 大きなディスプレイだと切り替えボタンが表示され、カラム表示の切り替えができる
- 切り替えの状態は保存され、トップページやトレンドでは同じ表示になる

<img width="842" alt="スクリーンショット 2022-03-29 23 50 31" src="https://user-images.githubusercontent.com/5019072/160639767-ad20b2a7-f11d-452b-a071-f3f9bf2d9e20.png">

2カラムにすることによって、記事カードがごちゃっと見えるものもあるのでそのあたり調整は必要かもしれません。

**Comment by kyntk** [comment]
@yamadashy ご意見ありがとうございます！
2カラム表示を気に入っていただけて嬉しいです！:heart: 

2カラムにして良かったという声もいただいたため、カスタムできる機能も個人的にも良さそうかと思います。
引き続きご意見をいただきながらUIの検討していきます！

**Comment by yamadashy** [comment]
ご返信ありがとうございます！

ご検討宜しくお願いいたします 🙇🏻
